docs/markdown/Introduction/how-does-pants-work.md

```diff
@@ -68,6 +68,7 @@ Work is broken down into many small units and kept warm in a daemon so that as l
 ### Hermetic execution
 
 Pants sandboxes all processes that it executes, ensuring that cache keys are always accurate, and builds are always correct.
+This implies, amongst other things, processes can't access your local [environment variables](docs:reference-subprocess-environment) or even setting stored in your home directory (as this uses `HOME`).
```

Comment:
The `subprocess-environment` references are only good for the Python backend today; so they risk sounding authoritative and yet being off-point. I think the best you can do is to encourage looking for where / how to pass environment variables for the context you're in. Maybe give the `subprocess-environment` as a qualified example of how you can do this for Python. N.B. that its not even always the right thing to do for Python! If the env var is only needed for tests, you can reduce scope with https://www.pantsbuild.org/docs/reference-test#extra_env_vars - and that works for any language stack, including Python.

Which is all just to say the concept is clear, the details are all over the place.

Reply:
Also worth noting is that for the tools Pants integrates with, config files from your `HOME` do get picked up where applicable without any additional configuration needed.